src/transformers/modeling_rag.py

```diff
@@ -600,12 +604,16 @@ def forward(
             doc_scores is not None
         ), "Make sure that `doc_scores` are passed when passing `encoder_outputs` to the forward function."
 
+        assert (
+            doc_scores.shape[1] % n_docs
+        ) == 0, "Make sure `doc_scores.shape[1]` is always have a size of `n_docs`."
```

Comment:
```suggestion
        ) == 0, f" The first dimension of `context_input_ids` should be a multiple of `n_docs`={n_docs}, but is {context_input_ids.shape[0]}."
```

Reply:
@patrickvonplaten Please review your this change change, while debugging I have seen `context_input_ids == None` at this point. Specially while using `RagSequenceForGeneration`  without `retriever`. Hence it should be -
```
f" The first dimension 1 of `doc_scores` should be a multiple of `n_docs`={n_docs}, but is {doc_scores.shape[1]}."
```